infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackService.kt

```diff
@@ -666,6 +667,11 @@ class LocationTrackService(
         return dao.fetchLocationTrackOwners()
     }
 
+    fun getLocationTrackOwner(id: IntId<LocationTrackOwner>): LocationTrackOwner {
+        return getLocationTrackOwners().find { owner -> owner.id == id }
+            ?: throw NoSuchEntityException(LocationTrackOwner::class, id)
+    }
+
```

Comment:
Nyt iteroidaan lista läpi, mutta ovat ainakin toistaiseksi kaikki 97kpl muistissa, joten ei pitäisi viedä suuresti aikaa.

Reply:
Tämä on nähdäkseni ihan ok. Näitä on tosiaan kuitenkin vain vähän, ja ne kaikki haetaan yhdellä kanta-round-tripillä -> ei pitäisi olla perffiongelmia. Voidaan säätää sitten jos joskus tilanne muuttuu